main.go

```diff
@@ -119,6 +122,10 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Database")
 		os.Exit(1)
 	}
+	if err = (&kcirocksv1beta1.Database{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
commenting out this part should simply to disable webhook

Reply:
is the webhook mgr running in at the same time in the main db-operator?

Other project, has a dedicated pod for this. Should we also split for better log seperation and issue investigation?

Reply:
I was thinking about it. But I'm more in favour of using the same pod, but another container

Reply:
@dabde I'm afraid that if we decide to separate webhook and operator, this issue will not be closed any time soon. Maybe we can put them in the same container now and create an issue to separate later?

Reply:
yeah, it fine.